packages/api/src/contexts/ApiContext.test.tsx

```diff
@@ -0,0 +1,7 @@
+import { ApiProvider } from "./ApiContext";
+
+describe("ApiProvider", () => {
+  it("is truthy", () => {
+    expect(ApiProvider).toBeTruthy();
+  });
+});
```

Comment:
Ooh blijkbaar stond deze nog niet in een /src/tests folder. Zou je dat kunnen doen, dan is het in alle packages hetzelfde.